src/java.xml.crypto/share/classes/org/jcp/xml/dsig/internal/dom/Utils.java

```diff
@@ -47,20 +46,7 @@ private Utils() {}
     public static byte[] readBytesFromStream(InputStream is)
         throws IOException
     {
-        try (ByteArrayOutputStream baos = new ByteArrayOutputStream()) {
-            byte[] buf = new byte[1024];
-            while (true) {
-                int read = is.read(buf);
-                if (read == -1) { // EOF
-                    break;
-                }
-                baos.write(buf, 0, read);
-                if (read < 1024) {
-                    break;
-                }
-            }
-            return baos.toByteArray();
-        }
+        return is.readAllBytes();
```

Comment:
This is also from Apache Santuario. It's better to keep it unchanged.

Reply:
reverted